classes/assessment/assessment.php

```diff
@@ -196,6 +220,19 @@ public function check_assessment_validity(): \stdClass {
         return $result;
     }
 
+    /**
+     * Check if the assessment is valid for EC or SORA extension.
+     *
+     * @return bool
+     */
+    public function is_valid_for_extenstion(): \stdClass {
```

Comment:
Typo in extension

Reply:
Also, function declares return type `\stdClass`, PHPDoc says `@return bool`.

Reply:
Updated.